lib/ddtrace/contrib/sinatra/tracer_middleware.rb

```diff
@@ -8,46 +8,76 @@ module Contrib
     module Sinatra
       # Middleware used for automatically tagging configured headers and handle request span
       class TracerMiddleware
-        def initialize(app)
+        # Span resource when the application does not have the matching route
+        # NO_ROUTE_RESOURCE = 'no route'.freeze
```

Comment:
Is this something we plan to use down the line? If not, should remove

Reply:
Nah, I just forgot to remove it! Thank you for noticing!